packages/rocketchat-i18n/i18n/de.i18n.json

```diff
@@ -1065,6 +1065,7 @@
   "join-without-join-code_description": "Berechtigung, Kanäle mit Zutrittscode auch ohne Code zu betreten",
   "join-without-join-code": "Ohne Code beitreten",
   "join": "Beitreten",
+  "Join-room-request": "__user__ möchte diesem Raum als Mitglied beitreten.  Besitzer oder Moderator sollten darauf antworten",
```

Comment:
All these translations are relevant to this feature and should be contributed back to RC. so no change required.